corehq/apps/locations/forms.py

```diff
@@ -241,6 +246,12 @@ def clean_site_code(self):
     def clean_location_type(self):
         loc_type = self.cleaned_data['location_type']
 
+        try:
+            loc_type_obj = LocationType.objects.get(Q(code=loc_type)|Q(name=loc_type))
```

Comment:
i guess this is probably related to the above question (but seems a bit weird to me). why do we check both fields?